### DISCUSSION TITLE
Error executing go-aws-sso

### DISCUSSION THREAD

**Discussion by sagisv** [theurichde/go-aws-sso#157] [Q&A] [answered]
Have installed go-aws-sso, when trying to execute go-aws-sso its throwing up the following error: Error while initializing logger: sync /dev/stdout: inappropriate ioctl for device; sync /dev/stderr: inappropriate ioctl for device).

Please provide any suggestions to resolve this

Thank you in advance !!

**Comment by theurichde** [comment, marked as answer]
Hi @sagisv  - please try updating to [v1.4.2](https://github.com/theurichde/go-aws-sso/releases/tag/v1.4.2) - that issue should be fixed with this version. 

Let me know if that helped :-)

**Reply by sagisv** [reply]
Hi @theurichde - I just updated with the latest version and it fixed the issue.

Thank you so much
